core/controllers/reader_test.py

```diff
@@ -182,6 +182,38 @@ def test_give_feedback_handler(self):
         self.logout()
 
 
+class ExplorationHandlerTest(test_utils.GenericTestBase):
+    """Test the handler for providing the initial data
+    for a single exploration.
+    """
+
+    def test_error_if_exploration_is_none(self):
```

Comment:
Actually, can't you add this test function to the already existing class?

Reply:
I'm not really sure which test class I should put it under, since none of them seem to focus on this class. Do you have any suggestions?

Reply:
@aks681 Sorry for the ping, but I thought I would bump this in case you missed it.